src/components/ArticleCollection.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from "react";
+import Articles from "../modules/Articles";
+import useStyles from "../styles/styles";
+import {
+  Typography,
+  Container,
+  CardContent,
+  Grid,
+  Card,
+} from "@material-ui/core";
+
+const ArticleCollection = () => {
+  const [articles, setArticles] = useState([]);
+  const classes = useStyles();
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const data = await Articles.index();
+      setArticles(data);
+    };
+    fetchData();
+  }, []);
+
+  return (
+    <div>
+      <Container className={classes.cardGrid}>
+        <Grid container spacing={4}>
+          {articles.map((article) => (
```

Comment:
You don't do a .map() here... You can have a look at previous projects done as a reference.